record-util/src/main/java/org/hps/record/daqconfig2019/DAQConfig2019Driver.java

```diff
@@ -329,6 +339,62 @@ else if((runNumber >= 10637 && runNumber <= 10644) || (runNumber <= 10647 && run
             return "hps_v12_1";   
         else if(runNumber >= 10716 && runNumber <= 10718)
             return "hps_v13_FEE";
+        

```

Comment:
Just to be clear for TongTong, my comment was not about the data from the config files being a resource. My comment was about this particular block of code being a mess. What this code achieves can be more cleanly achieved via the db, by making a table that holds the names of each of the config file resources that could be used and making entries in the conditions db to associate each one with the appropriate run numbers.